CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-### Unreleased
+### 2.15.1 2019-03-14
```

Comment:
It's really noddy but can you add a new `### Unreleased` line 2 lines above this?